### ISSUE TITLE
Cannot install apc when php is installed from source

### ISSUE THREAD

**Issue by roylindauer** [sous-chefs/php#86]
I am installing php from source (5.4.32) on Centos 6.5. I then try to run the following:

```
php_pear "apc" do
    action :install
end
```

Which appears to be successful (no errors reported), however apc.so is nowhere to be found on my system. 

I tried to install it manually by running `sudo pecl install apc`, and it compiles fine, but again, apc.so is nowhere to be found. It appears that it is installing the extension under `/tmp/pear/temp/pear-build-rootJlliUZ/install-APC-3.1.13` but that directory ceases to exist once the install is complete. 

The extension dir in php.ini: `/usr/lib64/php/modules`
Additional .ini files parsed: `/etc/php.d/apc.ini`

**Comment by Xorima** [comment]
Closing due to inactivity. 

If this is still an issue please reopen or open another issue. Alternatively drop by the #sous-chefs channel on the [Chef Community Slack](http://community-slack.chef.io/) and we'll be happy to help! 

Thanks,
Sous-Chefs
